package.json

```diff
@@ -81,10 +83,12 @@
     "prism-react-renderer": "2.4.0",
     "react": "18.3.1",
     "react-dom": "18.3.1",
+    "tinybench": "^2.9.0",
```

Comment:
do we need tinybench => did we switch to vitest?

Reply:
Good catch, this is from an earlier version. Vitest wraps tinybench internally, so no need to pin it explicitly.